portforward.sh

```diff
@@ -19,16 +20,20 @@ if [ "$json" == "" ]; then
     printf "Port forwarding is already activated on this connection, has expired, or you are not connected to a PIA region that supports port forwarding\n"
     exit 1
 fi
-port=`echo $json | grep -Eo [0-9]{3,5}`
```

Comment:
I would rather keep it to using grep to reduce the size and have less dependencies. The json answer is quite simple so no need for jq I think.

Reply:
`jq` is a really common package, it just adds 22k to the image and no other dependencies.

Parsing Json with `grep` is really really weak, should PIA add a space or another element in the hash and you script is broken.

I like small images too, but I would not push the zeal to the point of using broken script to save a few kB...

Reply:
Ok fine, let's keep jq onboard then.